x-pack/plugins/reporting/public/share_context_menu/register_pdf_png_reporting.tsx

```diff
@@ -116,93 +77,67 @@ export const reportingScreenshotShareProvider = ({
     const { sharingData } = shareOpts as unknown as { sharingData: ReportingSharingData };
     const shareActions = [];
 
-    const pngPanelTitle = i18n.translate('xpack.reporting.shareContextMenu.pngReportsButtonLabel', {
-      defaultMessage: 'PNG Reports',
-    });
-
-    const jobProviderOptions: JobParamsProviderOptions = {
+    const jobProviderOpts: JobParamsProviderOptions = {
       shareableUrl: isDirty ? shareableUrl : shareableUrlForSavedObject ?? shareableUrl,
       objectType,
       sharingData,
     };
 
-    const isV2Job = isJobV2Params(jobProviderOptions);
+    const isV2Job = isJobV2Params(jobProviderOpts);
     const requiresSavedState = !isV2Job;
+    const reportingModalTitle = i18n.translate('xpack.reporting.shareContextModal.buttonLabel', {
+      defaultMessage: 'Exports',
+    });
 
-    const pngReportType = isV2Job ? 'pngV2' : 'png';
-
-    const panelPng = {
-      shareMenuItem: {
-        name: pngPanelTitle,
-        icon: 'document',
-        toolTipContent: licenseToolTipContent,
-        disabled: licenseDisabled || sharingData.reportingDisabled,
-        ['data-test-subj']: 'PNGReports',
-        sortOrder: 10,
-      },
-      panel: {
-        id: 'reportingPngPanel',
-        title: pngPanelTitle,
-        content: (
-          <ScreenCapturePanelContent
+    const openImageModal = () => {
+      const session = overlays.openModal(
+        toMountPoint(
+          <ReportingModalContent
+            onClose={() => {
+              session.close();
+            }}
             apiClient={apiClient}
             toasts={toasts}
             uiSettings={uiSettings}
-            reportType={pngReportType}
             objectId={objectId}
             requiresSavedState={requiresSavedState}
-            getJobParams={getJobParams(apiClient, jobProviderOptions, pngReportType)}
             isDirty={isDirty}
-            onClose={onClose}
             theme={theme}
-          />
+            jobProviderOptions={jobProviderOpts}
```

Comment:
Could this be `getJobParams={getJobParams(apiClient, jobProviderOptions, pngReportType)}` for consistency with historical usage, and the CSV modal?
```suggestion
            getJobParams={getJobParams(apiClient, jobProviderOptions, pngReportType)}
```

I hope this might smooth issues with https://github.com/elastic/kibana/pull/171253/files#r1442318739

Reply:
I am trying to figure out how to get only one panel for PDF and PNGs and the pdfReportType and pngReportType needing to be passed into getJobParams() is throwing me off on how to pass only one panel to the share provider...